logisland-components/logisland-processors/logisland-processor-common/pom.xml

```diff
@@ -127,6 +127,12 @@
             <artifactId>delight-nashorn-sandbox</artifactId>
             <version>0.1.15</version>
             <optional>true</optional>
+            <exclusions>
```

Comment:
Why you needed this ? Would be cool to add a comment

Reply:
not relevant in this PR, can be moved into a "dependency cleanup issue"

Reply:
I am sorry but for me it is relevant. The change was made in this PR, in develop branch there is no exclusion. You added this exclusion in your first commit "fix plugin loader in jdk12" so I guess this is related to jdk 12 compatibility ? Please add a comment about those exclusion in pom files otherwise we will forgot we added this because for jdk 12 compatibility.